docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -137,6 +137,8 @@ Property                                              Description
 ``bigquery.credentials-key``                          The base64 encoded credentials key                             None. See the `requirements <#requirements>`_ section.
 ``bigquery.credentials-file``                         The path to the JSON credentials file                          None. See the `requirements <#requirements>`_ section.
 ``bigquery.case-insensitive-name-matching``           Match dataset and table names case-insensitively               ``false``
+``bigquery.query-cache.enabled``                      Enable `query cache
```

Comment:
nit: would query-results-cache be better?

Reply:
Do you mean `bigquery.query-results-cache.enabled`? or just `bigquery.query-results-cache`?

Reply:
`bigquery.query-results-cache.enabled`.